wms_connector/models/synchronize_exportable_mixin.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2023 Akretion
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import base64
+import csv
+import datetime
+from io import StringIO
+
+from odoo import fields, models
+from odoo.tools import config
+
+
+class SynchronizeExportableMixin(models.AbstractModel):
+    _name = "synchronize.exportable.mixin"
+    _description = "Synchronizable export mixin"
+
+    wms_export_date = fields.Datetime(readonly=True)
+    wms_export_attachment_id = fields.Many2one(
+        "attachment.queue", index=True, readonly=True
+    )
+    wms_export_error = fields.Char(readonly=True, index=True)
```

Comment:
This mixin is used in pickings, OK. A picking is either wms exportable or wms importable, isn't it ?

If yes fields names should be `wms_date`, `wms_attachment_id`, `wms_error` ou wms_exception.

Qu'en pensez vous ?